packages/server/src/datasources/dataset-service.ts

```diff
@@ -3,7 +3,6 @@ import * as path from "path";
 import * as fs from "fs";
 import { createWriteStream, unlink } from "fs";
 import * as mkdirp from "mkdirp";
-import * as tar from "tar";
```

Comment:
This is used on line 161, to extract the tar. It is currently commented out to not break ZIP files, but this import statement is needed for that